index.html

```diff
@@ -35,12 +35,19 @@ <h2>Grow your garden</h2>
     <div class="products">
         <center><h2>Plant native and invest in new Zealand's future</h2></center>
         <h2 id="h2padding">Our Hebe's</h2>
-        <div class="imgBox">
-            <img src="images/FreshLogo4.png" width=20%>
-            <img src="images/FreshLogo4.png" width=20%>
-            <img src="images/FreshLogo4.png" width=20%>
-            <img src="images/FreshLogo4.png" width=20%>
-            
+        <div class="imgBoxRow">
+            <div class="imgBoxColumn">
+                <img src="images/FreshLogo4.png" height=100%>
+            </div>
+            <div class="imgBoxColumn">
+                <img src="images/FreshLogo4.png" height=100%>
+            </div>
+            <div class="imgBoxColumn">
+                <img src="images/FreshLogo4.png" height=100%>
+            </div>
+            <div class="imgBoxColumn">
+                <img src="images/FreshLogo4.png" height=100%>
+            </div>
```

Comment:
I removed the old system for image placement and replaced it with the grid method for greater flexibility.
From testing I have decided that the grid system is the best for placing images but the horisontal layout 
(from v1 ) works best.